test/e2e/traceflow_test.go

```diff
@@ -27,16 +27,23 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 
 	"github.com/vmware-tanzu/antrea/pkg/apis/ops/v1alpha1"
+	"github.com/vmware-tanzu/antrea/pkg/ops"
 )
 
-const traceflowTimeout = 40 * time.Second
+const traceflowTimeout = 80 * time.Second
```

Comment:
since this is a pretty large timeout, maybe it would be nice to have a comment explaining why?

Reply:
I removed this total timeout and use a 15 second timeout for each Get-traceflow.